docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[synthetics-params-secrets]]
-= Working with params and secrets
+= Working with parameters and secrets
```

Comment:
I'm not familiar with this content, but I wonder if this should be `params` too?